GoogleForCreators/web-stories-wp
# Fix updating page advancement on the first load

## Summary



## Relevant Technical Choices



## To-do




## User-facing changes



## Testing Instructions
1. Add a new Story
2. Verify that "Save draft" button is not active without making any changes
3. Switch to the Document panel
4. Verify that "Save draft" button doesn't change to active.




---



Fixes #4094 
Fixes #1907
